packages/client-core/src/user/components/Wallet/wallet.tsx

```diff
@@ -115,9 +115,9 @@ export const WalletPage = (): any => {
     }))
     try {
       const response = await client.service('user-inventory').patch(data[0].user_inventory.userInventoryId, {
-        privateKey: walletData[0].privateKey,
-        fromUserAddress: walletData[0].userAddress,
-        toUserAddress: walletDataReceive[0].userAddress,
+        //privateKey: walletData[0].privateKey,
+        //fromUserAddress: walletData[0].userAddress,
+        //toUserAddress: walletDataReceive[0].userAddress,
         quantity: Number(data[0].user_inventory.quantity) - Number(amt),
         walletAmt: Number(amt),
         type: 'Wallet transfer'
```

Comment:
lets rename this from `Wallet transfer` to `transfer`

Reply:
Changed in the current version